cms/djangoapps/contentstore/outlines.py

```diff
@@ -205,7 +302,7 @@ def get_outline_from_modulestore(course_key) -> Tuple[CourseOutlineData, List[Co
     content_errors = []
 
     with store.branch_setting(ModuleStoreEnum.Branch.published_only, course_key):
-        course = store.get_course(course_key, depth=2)
+        course = store.get_course(course_key, depth=3)
```

Comment:
Was this change made to get units as well as sequentials? Whatever the reason, a comment would be helpful. Or maybe `UNIT_LEVEL_DEPTH=3` elsewhere - and use `depth=UNIT_LEVEL_DEPTH` here?